### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by bas-ie** [rrousselGit/riverpod#2779] [General]
This is a generic, "thanks for all the hard work" message to interleave with the support requests! I'm really enjoying the way Riverpod has made my life easier, converting some fairly ugly stateful components into much tidier abstractions.

Our household doesn't quite have enough disposable income for a monthly sponsorship at the moment, but I've dropped a beer's worth in the one-off sponsorship. :heart:

**Comment by catalunha** [comment]
I follow your same path.
Thanks to Remi and the entire team for their care and attention to every detail of the library and documentation. It is now building a cookbook to facilitate and expand usage.

**Comment by rrousselGit** [comment]
Thanks! It means a lot to me :)
